### PR TITLE
CW-331-Add-option-to-disable-market-place-in-display-settings

### PR DESCRIPTION
**Pull request by Blazebrain**
Issue Number (if Applicable): Fixes #

# Description
Add switcher in display settings to enable/disable marketplace page

if enabled: it will be the same behavior as it is right now

if disabled: the dashboard screen will only have the balance and transactions screen

# Pull Request - Checklist  

- [x] Initial Manual Tests Passed
- [] Double check modified code and verify it with the feature/task requirements
- [x] Formate code
- [x] Look for code duplication
- [x] Clear naming for variables and methods

### CODE REVIEW

**Review by OmarHatem28 — CHANGES_REQUESTED** [review]
also, you haven't added a reaction for `shouldShowMarketPlaceInDashboard` to save its new value in the shared prefs.
and please revert the formatting in `settings_store` and in `dashboard_page`

File: lib/store/settings_store.dart [hunk 1]
```diff
@@ -232,62 +238,74 @@ abstract class SettingsStoreBase with Store {
...
             .getInt(PreferencesKey.currentBalanceDisplayModeKey)!);
     // FIX-ME: Check for which default value we should have here
-    final shouldSaveRecipientAddress =
-        sharedPreferences.getBool(PreferencesKey.shouldSaveRecipientAddressKey) ?? false;
+    final shouldSaveRecipientAddress = sharedPreferences
+            .getBool(PreferencesKey.shouldSaveRecipientAddressKey) ??
+        false;
+    final shouldShowMarketPlaceInDashboard = sharedPreferences
+            .getBool(PreferencesKey.shouldShowMarketPlaceInDashboard) ??
+        false;
```
[the remark this answers is not in the document]
Comment (Blazebrain) [reply lib/store/settings_store.dart]: Fixed this  @OmarHatem28

**Comment by Blazebrain** [comment]
> also, you haven't added a reaction for `shouldShowMarketPlaceInDashboard` to save its new value in the shared prefs to be persistent.
> and please revert the formatting in `settings_store` and in `dashboard_page`

Added the persistence also, and remove trailing commas format I added in the store and dashboard

**Comment by OmarHatem28** [comment]
Reviewed ok, please check Justin's comment on Jira

**Comment by SamsungGalaxyPlayer** [comment]
@OmarHatem28 tests ok now

**Review by OmarHatem28 — APPROVED** [review]

**Pull request merged**